### ISSUE TITLE
After "container init" standby sometimes hangs at initial wal segments

### ISSUE THREAD

**Issue by ghost** [emarsys/pgarchive#7]
The problem manifests in standby log repeating lines like the following endlessly:

```
2016-04-20 17:31:00.340 CEST,,,31825,,5717a0b4.7c51,6,,2016-04-20 17:31:00 CEST,,0,LOG,00000,"invalid record length at 0/56000140",,,,,,,,,""
2016-04-20 17:31:00.353 CEST,,,31825,,5717a0b4.7c51,7,,2016-04-20 17:31:00 CEST,,0,LOG,00000,"restored log file ""000000010000000000000056"" from archive",,,,,,,,,""
```

The wal_archive looks fine, and is streaming replication data, but closer inspection and comparison with another archive shows that an early segment file is corrupted.

**Comment by ghost** [comment]
The problem is most likely how the wal_archive is seeded with files from standby/pg_xlog in function 
container_init(). The last segment copied over should be renamed to *.partial, otherwise pg_receivexlog assumes it is complete.

The same problem may exist in container_resync().

A workaround is to stop wal_archive, find the broken segment, rm all newer ones, and rename it manually to *.partial, and then start wal_archive again. This only works as long as the upstream server still has all the wal files.

**Comment by ghost** [comment]
Fixed with #9.
